bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-assessment",
-  "version": "4.5.2",
+  "version": "4.5.3",
```

Comment:
can you leave these to @joe-allen-89 to handle when her gets to do releases? we often have conflicting version numbers that have to be resolved at release time anyway.

it's probably better to mark the pr and issue with 'breaking', 'enhancement' or 'bug' accordingly, so that whoever does the releases gets a better idea of what they're looking at